### ISSUE TITLE
Enhancement: Add a 2nd method of adding playable and custom weeks

### ISSUE THREAD

**Issue by Voidymite** [KadeArchive/Kade-Engine#1306]
## What is your suggestion, and why should it be implemented?
A 2nd method of adding custom weeks to the game. This method goes as follows:
Add the week itself. Make a line under Week 6's levels and add your levels. It should look like this:
['Ugh', 'Guns', 'Stress']
At the end of Week 6's levels, add a comma, like this:
['Senpai', 'Roses', 'Thorns'],
Add another true to this line:
public static var weekUnlocked:Array<Bool> = [true, true, true, true, true, true, true];
After adding another true, it should look like this:
public static var weekUnlocked:Array<Bool> = [true, true, true, true, true, true, true, true];
Do nearly the same you did last time with the character for the week. It should look like:
['senpai', 'bf', 'gf'],
['tankman', 'bf', 'gf']
If you don't have a custom character, set it to 'dad'.
Do the same thing, again:
"Hating Simulator ft. Moawling",
"TANKMAN"
Now that you've added your week code, find every line with "trace" in it and add 3 to the value in it. For example:
Before:
trace("Line 24");
After:
trace("Line 27");
Add a story menu icon so you can see the week's story mode asset in the Story Mode menu.

You're done! This tutorial, though, is only for adding weeks. You cannot use custom backgrounds with this tutorial unless you make a week folder and code all and draw all of the background assets.

**Comment by isakube** [comment]
This is literally the exact same procedure we have documented.

**Comment by Lucky-56** [comment]
no, since I changed the week unlocking but for the rest: **yes**

**Comment by Voidymite** [comment]
> This is literally the exact same procedure we have documented.

this was kind of based off of an idea i had when i saw a recent commit, i didnt think it already existed. still, the week unlocking code isnt in the original tutorial lol

**Comment by isakube** [comment]
> > This is literally the exact same procedure we have documented.
> 
> this was kind of based off of an idea i had when i saw a recent commit, i didnt think it already existed. still, the week unlocking code isnt in the original tutorial lol

it is. read.
![Screen Shot 2021-07-14 at 6 46 40 PM](https://user-images.githubusercontent.com/68293280/125715064-bb78a2f0-7c85-4f87-8a02-baed04e3bf17.png)

**Comment by Lucky-56** [comment]
oh heh, reminder to close
